npm/private/npm_translate_lock.bzl

```diff
@@ -32,6 +35,10 @@ _ATTRS = {
     "verify_node_modules_ignored": attr.label(),
     "link_workspace": attr.string(),
     "bzlmod": attr.bool(),
+    "root_package": attr.string(default = DEFAULT_ROOT_PACKAGE),
```

Comment:
Would there be any value prefixing these with `_` or something like that just to further emphasize that they are internal? (Actually... does bazel allow that?).

Reply:
`_` prefix makes them private so they can't be set from outside

Reply:
in could prefix them with something `internal_` but then when they are hoisted to the public API it is breaking for downstream rule sets

Reply:
the contract of undocumented means private API is reasonable

Reply:
Yeah I think the `_`/private + breaking for downstream rulesets makes it worth keeping the names as-is, just no telling people that they exist 😅